app/controllers/api/v1/tickets_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class TicketsController < ApiController
+      def create
+        # Checks if the email is registered to an admin user
```

Comment:
if you need comments, it's because the method is not pretty clear..
I suggest you to move the line below to a comment that check if the email is already in use.. or something like that.

```
def email_already_in_use?(email)
  AdminUser.find_by(email: params[:email]).blank?
end 
```

Reply:
Also avoid having more than 100 lines per row